alpha/nodejs-chart/README.md

```diff
@@ -0,0 +1,73 @@
+```
+
+You can use this secret by passing `--set build.pullSecret=my-pull-secret` to `helm install`, or you can configure this in a values file:
+```yaml
+build:
+  pullSecret: my-pull-secret
+```
+and apply by passing `--values $VALUES_FILE`.
+
+### Push Secret
+You will need to create a push secret if you want to push your image to an external registry. Use the following command as a reference to create your push secret:
+```bash
+oc create secret docker-registry my-push-secret --docker-server=$SERVER_URL --docker-username=$USERNAME --docker-password=$PASSWORD --docker-email=$EMAIL
+```
+
+You can use this secret by passing `--set build.output.pushSecret=my-push-secret` and `--set build.output.kind=DockerImage` to `helm install`, or you can configure these in a values file:
+```yaml
+build:
+  output:
+    kind: DockerImage
+    pushSecret: my-push-secret
+```
+
+## Values
+This section describes the Values used to configure this chart.
+
+Below is a table of each value used to configure this chart.
+
+| Value | Description | Default | Additional Information |
+| ----- | ----------- | ------- | ---------------------- |
+| `image.name` | Name of the image you want to build/deploy | Defaults to the Helm release name. | The chart will create/reference an [ImageStream](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html) based on this value. |
+| `image.tag` | Tag that you want to build/deploy | `latest` | The chart will create/reference an [ImageStreamTag](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html#images-using-imagestream-tags_image-streams-managing) based on the name provided |
+| `build.enabled` | Determines if build-related resources should be created. | `true` | Set this to `false` if you want to deploy a previously built image. Leave this set to `true` if you want to build and deploy a new image. |
+| `build.uri` | Git URI that references your git repo | https://github.com/nodeshift-starters/nodejs-rest-http | This value defaults to a sample application. Be sure to override this if you want to build and deploy your own application. |
+| `build.ref` | Git ref containing the application you want to build | main | - |
+| `build.contextDir` | The sub-directory where the application source code exists | - | - |
+| `build.output.kind` | Determines if the image will be pushed to an ImageStreamTag or a DockerImage (external registry) | ImageStreamTag | More information: More information: https://docs.openshift.com/container-platform/4.6/builds/managing-build-output.html |
+| `build.output.pushSecret` | Push secret name | - | Used only if build.output.kind == 'DockerImage' |
+| `build.pullSecret` | Image pull secret | - | More information: https://docs.openshift.com/container-platform/4.6/openshift_images/managing_images/using-image-pull-secrets.html |
```

Comment:
@AshCripps Can you add `build.output.kind` and `build.output.pushSecret` to the values.schema.json also? Apologies, missed this earlier. 

Here's an example from the Quarkus chart - https://github.com/redhat-developer/redhat-helm-charts/blob/master/alpha/quarkus-chart/values.schema.json#L80-L94